tests/integration/testlib/assertlib.py

```diff
@@ -29,5 +32,39 @@ def assert_state(desired_state_data):
 
     full_desired_state = statelib.State(current_state.state)
     full_desired_state.update(desired_state_data)
+    _ignore_ipv6_link_local(current_state.state, full_desired_state.state)
+    _sort_ip_addresses(current_state.state, full_desired_state.state)
```

Comment:
You can add it now to State.normalize (after you rebase).

I do not know if the "ignore" one will fit as a method there (under normalize or other).